tests/functional/kata-deploy/kata-deploy.bats

```diff
@@ -82,7 +72,18 @@ setup() {
 	fi
 
 	local cmd="kubectl -n kube-system get -l name=kata-deploy pod 2>/dev/null | grep '\<Running\>'"
-	waitForProcess 240 10 "$cmd"
+
+	if ! waitForProcess 240 10 "$cmd"; then
+		echo "Kata-deploy pod is not running. Printing pod details for debugging:"
+		kubectl -n kube-system get pods -o wide
+		kubectl -n kube-system get pods -l name=kata-deploy -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | while read -r pod; do
+			echo "Describing pod: $pod"
+			kubectl -n kube-system describe pod "$pod"
+		done
+		echo "ERROR: kata-deploy pod is not running, tests will not be execute."
+		echo "ERROR: setup() aborting tests..."
+		return 1
+	fi
```

Comment:
What is the extra code here needed for? It is merge conflict that brought it back?

Reply:
This deserves its own commit, I'm just increasing the debug messages logs in case of failure, because it was not possible to by the logs what was wrong. I will split this to a different commit.